crocket/src/main/java/com/crocket/IEventListener.java

```diff
@@ -0,0 +1,5 @@
+package com.crocket;
+
+public interface IEventListener {
+    public void handleEvent(PassHoopEvent event);
```

Comment:
What does the name PassHoopEvent come from? Is the interface only for passing hoops?

Reply:
It has been renamed to PassTargetEvent